internal/strategy/customstrategies.go

```diff
@@ -0,0 +1,34 @@
+package strategy
```

Comment:
moved `strategy/` and `transfer/` to main folder. Removed `internal/` folder.

Reply:
AFAICS strategy package is only used by `sync.go` and I don't see any other future use case for this package. Moving this codebase (only 34 loc) to sync.go would be a better option. What do you think?

Reply:
I think current approach is better. Because
1. We can seperate the comparsion to another interface and if we would like to use another comparison function (which includes hashes etc.) we just need a struct which implements `Compare` function.
2. As far as I know, `cp` command has a `shouldOverride` function. We can change it to `strategy` interface, the code would be much more cleaner. So, it can be used in another parts as well. Therefore, I created it as another interface.

Reply:
It's good practice to move code into its own package if it improves readability and testability. OTOH, I agree with @sonmezonur and @seruman. `transfer` package is not needed. We can move relevant methods into `sync.go`.

For the sync strategy code, it's good it's behind an interface. But the package is not needed. We can again move it under the `command` package with its own file;  may be a `sync_strategy.go`.